features/debug-tools/rules.mk

```diff
@@ -0,0 +1,4 @@
+GDB_PYTHON_VERSION=$(shell $(FEATURESDIR)/debug-tools/bin/get-gdb-python-version gdb)
+
+PUT_FEATURE_PROGS += $(DEBUG_TOOLS_PROGS)
+PUT_FEATURE_FILES += $(DEBUG_TOOLS_FILES) $(LIBDIR)/python$(GDB_PYTHON_VERSION)
```

Comment:
are you sure about `$(LIBDIR)/python$(GDB_PYTHON_VERSION)` ?
It looks like you want to pack `sys.path`.